src/js/queries/globalGuidePagesQuery.js

```diff
@@ -1,17 +0,0 @@
-import guidePageFragment from './guidePageFragment';
```

Comment:
What were these, where are they now? How is deleting these related to adding the Form Page? Or is it just updating things from the other stuff @briaguya merged in?